selfdrive/car/gm/carcontroller.py

```diff
@@ -30,7 +30,6 @@ def __init__(self, dbc_name, CP, VM):
     self.cancel_counter = 0
 
     self.lka_steering_cmd_counter = 0
-    self.sent_lka_steering_cmd = False
```

Comment:
this is not a good assumption. you can add valid/alive flag to the can parser if you want instead

Reply:
you can also just check canValid

Reply:
the old assumption wasn't good or this one?

Reply:
made it more explicit by only updating CS's ts_nanos on explicit update of the message